File: TicketWeb/src/app/models/LoginResponse.ts
import { Agent } from 'src/app/models/Agent';
import { TicketsInfo } from './TicketsInfo';
export class LoginResponse {
    TicketSystemInfo: Array<TicketsInfo>;

    constructor() {
        this.TicketSystemInfo = new Array<TicketsInfo>();
    }
}

File: TicketSystem.API/Models/LoginResponse.cs
using System.Collections.Generic;

namespace TicketSystem.API.Models
{
    public class LoginResponse
    {
        public List<TicketsInfo> TicketSystemInfo { get; set; }
        public LoginResponse()
        {
            TicketSystemInfo = new List<TicketsInfo>();
        }
    }
}
